### DISCUSSION TITLE
Grub hide option

### DISCUSSION THREAD

**Discussion by sajjad000** [linuxdeepin/developer-center#3498] [Features Request & Ideas | 特性请求 & 头脑风暴]
Please add an option to hide the grub menu on startup because I want deepin to be default without seeing the grub menu

**Comment by BLumia** [comment]
cc @meiyixiang

**Comment by raspin0** [comment]
![obraz](https://user-images.githubusercontent.com/39202451/205439685-77a771e5-f34b-4db6-87fb-86c83a7b9040.png)

There's an option to reduce waiting time from 5 to 1 second. You can install `Grub Customizer` from the App Store to remove delay entirely, however if your system breaks you might have a hard time to fixing it and entering grub.
